apps/www/src/public/registry/index.json

```diff
@@ -130,6 +130,22 @@
     ],
     "type": "components:ui"
   },
+  {
+    "name": "carousel",
+    "dependencies": [],
+    "registryDependencies": [
+      "utils"
+    ],
+    "files": [
+      "ui/carousel/Carousel.vue",
+      "ui/carousel/CarouselContent.vue",
+      "ui/carousel/CarouselItem.vue",
+      "ui/carousel/CarouselNext.vue",
+      "ui/carousel/CarouselPrev.vue",
+      "ui/carousel/index.ts"
+    ],
```

Comment:
Can you include missing file `useCarousel.ts` and move `interface.ts` content inside `index.ts`?

Reply:
Ok. I'll do it.

Reply:
Why moving `interface.ts` content to `index.ts` tho?